### PR TITLE
Replace VAR.name -> VAR.fullname and add VAR.shortname

### PR DESCRIPTION
**Pull request by leandro-lucarella-sociomantic**
New variables, available in package definition files.

`VAR.fullname` replaces the old `VAR.name` to make it more explicit, `VAR.shortname` contains only the package name, without the `VAR.suffix`.

### CODE REVIEW

**Comment by leandro-lucarella-sociomantic** [comment]
ping

**Comment by nemanja-boric-sociomantic** [comment]
Breaking change and no deprecation period? I guess it's fine for something in experimental stage, but just to check.

**Comment by leandro-lucarella-sociomantic** [comment]
What do you mean, is not a breaking change, `VAR.name`, while undocumented, will keep working... :)

**Comment by leandro-lucarella-sociomantic** [comment]
Maybe the *replace* is misleading

File: Makd.mak
Comment (leandro-lucarella-sociomantic) [inline Makd.mak:458]: See? `name` is still there...

Comment (nemanja-boric-sociomantic) [reply Makd.mak:458]: Yes, _replace_ is misleading :-)

**Review by nemanja-boric-sociomantic — APPROVED** [review]

**Comment by nemanja-boric-sociomantic** [comment]
In the lacks of tests, I will not merge this, feel free to automerge once you're confident it works.

**Comment by leandro-lucarella-sociomantic** [comment]
@joseph-wakeling-sociomantic maybe help testing? My simple test was OK.

@nemanja-boric-sociomantic maybe you can try it out quickly too in your project?

**Comment by nemanja-boric-sociomantic** [comment]
I'll try it.

**Comment by nemanja-boric-sociomantic** [comment]
LGTM

**Comment by nemanja-boric-sociomantic** [comment]
LGTM

File: README.rst
Comment (joseph-wakeling-sociomantic) [inline README.rst:663]: Spelling: `convenience` ;-)

**Comment by joseph-wakeling-sociomantic** [comment]
Hah, you folks were too quick for me :-)  I'll try to make time to test this out some time in the next day or two.  Looks fine in principle, though (bar one typo).

**Comment by leandro-lucarella-sociomantic** [comment]
@nemanja-boric-sociomantic is the fastest in the whole west, even when he is being conservative :D

#54

**Comment by nemanja-boric-sociomantic** [comment]
I just didn't see the typo :-)

**Pull request merged**
